src/modules/auth/ui/views/sign-up-views.tsx

```
@@ -69,6 +71,27 @@ export const SignUpView = () => {
     
   };
 
+  const onSocial=(provider: "github" | "google")=>{
+    setError(null);                                       {/*reset error*/}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)